Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/Dockerfile-torchserve

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.1-slim
+FROM python:3.7
```

Comment:
An idea for another PR (or maybe for #20 ), since we're so dependent on pickle for ser/des, would it make sense to have the same base image for both the trainer and the server so that there's much less chance of differences between them?
Another advantage of this is that the `pytorch/serve` repo's `install_dependencies.py` is pretty smart and will install the correct version of pytorch based on the system and cuda version available: https://github.com/pytorch/serve/blob/master/ts_scripts/install_dependencies.py#L25-L36, which means our trainer would have a more optimised version of pytorch based on where it spins up.

Reply:
By base image I mean we could do something like create a Dockerfile.base for installing python dependencies, then have two other files for the trainer and server that start FROM that base. Could be overkill but we've already had a lot of heartache with differences between train/serve

Reply:
> An idea for another PR (or maybe for #20 ), since we're so dependent on pickle for ser/des, would it make sense to have the same base image for both the trainer and the server so that there's much less chance of differences between them?

Definitely makes sense!